ElectronicObserver/Window/Settings/SubWindow/Browser/ConfigurationBrowserViewModel.cs

```diff
@@ -263,4 +266,20 @@ private void SelectScreenshotFolder()
 
 		ScreenShotPath = newSaveDataPath;
 	}
+
+	public static ValidationResult ValidateURL(string url, ValidationContext context)
+	{
+		if (context.ObjectInstance is not ConfigurationBrowserViewModel viewModel)
+		{
+			throw new NotImplementedException();
+		}
+		if (!Uri.IsWellFormedUriString(url, UriKind.Absolute) && !string.IsNullOrEmpty(url) && (Configuration.Config.FormBrowser.GadgetBypassServer == GadgetServerOptions.Custom))
```

Comment:
IsWellFormedUriString doesn't reject null to empty?
Also empty line between the ifs.